desktop/apps/consignments2/client/actions.js

```diff
@@ -578,6 +586,24 @@ export function updateLocationInputValues (city, state, country) {
   }
 }
 
+export function updateLocationFromSubmissionAndFreeze (city, state, country) {
```

Comment:
Since the location autocomplete handles separately from redux-form, we have to explicitly set the value and freeze the input when you return to the describe-your-work page.